package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@hs-web-team/eslint-config-ts",
-  "version": "0.5.0",
+  "version": "1.0.0",
   "description": "HubSpot Marketing WebTeam ESLint rules for Typescript",
   "main": "index.js",
   "engines": {
-    "node": ">=18"
+    "node": ">=18.18.0"
```

Comment:
Is there a reason of why the support for this specific version?

Reply:
It's a breaking change for one of the dependencies - [details here](https://typescript-eslint.io/blog/announcing-typescript-eslint-v7/#:~:text=Update%20Node.js%20engine%20requirement%20to%20%5E18.18.0%20%7C%7C%20%3E%3D20.0.0.%20This%20means%20we%20are%20dropping%20support%20for%20Node%2016%2C%2019%2C%20and%20Node%2018%20versions%20prior%20to%2018.18.0)

Reply:
**question:** Can we use this => `1.0.0` opportunity to just go to `20`?